test/integration/upload/cli/test_upload.py

```diff
@@ -60,7 +60,7 @@ def test_upload_with_dcp_type_option(self):
 
     @responses.activate
     @patch('hca.upload.s3_agent.S3Agent.upload_file')   # Don't actually try to upload
-    def test_no_transfer_acceleration_option_sets_up_botocore_config_correctly(self, upload_file_stub):
+    def test_no_transfer_acceleration_option_sets_up_botocore_config_correctly(self, *args):
```

Comment:
Is this because of a flake8 complaint? I think it would follow the style in the rest of the tests more if the `@patch` decorator got moved to the `with` block and the arguments got removed entirely, so like
```python
with patch('hca.upload.s3_agent.S3Agent.upload_file'), \
         patch('hca.upload.s3_agent.Config', new=Mock(wraps=botocore.config.Config)) as mock_config:
```

Reply:
No, actually this is because of a runtime error. Other suggestions?